test/parallel/test-repl-top-level-await.js

```diff
@@ -161,8 +161,10 @@ async function ctrlCTest() {
   ]), [
     'await timeout(100000)\r',
     'Thrown:',
-    'Error [ERR_SCRIPT_EXECUTION_INTERRUPTED]: ' +
-      'Script execution was interrupted by `SIGINT`',
+    '[Error [ERR_SCRIPT_EXECUTION_INTERRUPTED]: ' +
+      'Script execution was interrupted by `SIGINT`] {',
+    "  code: 'ERR_SCRIPT_EXECUTION_INTERRUPTED'",
+    '}',
```

Comment:
why did this test in particular need to change? is there something unusual about how this test works?

Reply:
The difference is that the `code` property became enumerable. That seems important so that users become aware that it's an actual property, not only part of `.stack`.

Reply:
In other words, because no other tests inspect `.stack` this way?

Reply:
It's not the stack that is changed here. It's the output when inspecting the error (e.g., using `console.log` or `util.inspect`). The reason is that only enumerable properties are inspected by default and others are ignored.

We rarely inspect errors in tests. Mostly we just check for the properties existence and their values. That also works with non-enumerable properties.